pkg/web/templates/partials/transaction/transaction_list.html

```diff
@@ -119,8 +119,10 @@
             <div tabindex="0" role="button" class="btn btn-ghost font-bold text-xl">&hellip;</div>
             <ul tabindex="0" class="dropdown-content z-[1] menu p-2 shadow bg-black font-semibold text-white rounded-box">
               <li><a href="/transactions/{{ .ID }}/info">View</a></li>
+              {{ if eq .Source "remote" }}
```

Comment:
This was removed during development while working on a previous update, so I've added this back.